Edit defence request from detail

novotnyjakub opened the pull request:
The Submitted defence request show page now has 2 links to "Change this". One for the detainee details and one in the Case details tab.

ansonK commented:
After updating Case details the page reloads showing the Interview tab.
Should it not reload with the Case details tab showing (as that was where the User was before) ?

novotnyjakub commented:
Good point, I'll take a look at that.

novotnyjakub commented:
OK @ansonK, this should be ready to go. I've added the support for coming back to the same tab. I've also refactored a bit some views and removed the `defence_request_management_spec` for good.

ansonK commented on app/controllers/defence_requests_controller.rb:
```diff
@@ -1,6 +1,7 @@
 class DefenceRequestsController < BaseController
 
   include DefenceRequestConcern
+  include DefenceRequestsHelper
```
Do we need to include a view helper into the controller?

novotnyjakub replied:
Yeah, so this is something I'm not sure about. Essentially `ApplicationHelper` is automatically available everywhere and I thought we're just splitting it semantically to smaller focused helpers. But if the purpose of all other helpers apart from `ApplicationHelper` is just views, than I'll move my url helper there.

ansonK commented:
:+1:
